.github/workflows/npm-publish.yml

```diff
@@ -25,6 +25,19 @@ jobs:
           # A file, directory or wildcard pattern that describes what to upload
           path: dist/*
           if-no-files-found: error
+      - name: Commit files
+      - uses: actions/checkout@v2
+        with:
+          persist-credentials: false # otherwise, the token used is the GITHUB_TOKEN, instead of your personal access token.
+          fetch-depth: 0 # otherwise, there would be errors pushing refs to the destination repository.
+        run: |
+          date > generated.txt
+          git config user.name github-actions
+          git config user.email github-actions@github.com
+          git add .
+          git commit -m "publish new dist 🎉 "
+          git push
+          branch: ${{ github.ref }}
```

Comment:
Is this supposed to be part of the run script or another property? I see the checkout actions doc has `ref` but not `branch`: https://github.com/actions/checkout

Reply:
I think this is not needed good 👁️

Reply:
I was going to use a different action before I realized checkout can do this